tensorflow-logging/tensorflow-logging.cabal

```diff
@@ -29,11 +29,11 @@ library
                 , stm
                 , stm-chans
                 , stm-conduit
-                , tensorflow == 0.2.*
-                , tensorflow-core-ops == 0.2.*
-                , tensorflow-ops == 0.2.*
-                , tensorflow-proto == 0.2.*
-                , tensorflow-records-conduit == 0.1.*
+                , tensorflow >= 0.2.0
```

Comment:
I don't think we tested the combinations, so it's probably safer to just change all the versions together and keep them at the same 0.3 value. WDYT?

Reply:
Since the haskell files (*.hs) were not altered when I patched for TF 2.3, it is safe to say that they are compatible with both the old and new version. However, the `proto-lens` dependency update (in order to move to a newer version of LTS) did cause a problem in terms of dependency conflict.

Doing this ensures that if someone wanted to patch an older version of the libraries with the new proto-lens package, they would be able to. However, I will admit that this is a pretty fringe case.

Thoughts?

Reply:
It's probably inconsequential either way. I'll merge as is.